modules/apps/remote-app/remote-app-service/service.xml

```diff
@@ -37,6 +37,7 @@
 		<column name="statusDate" type="Date" />
 	</entity>
 	<exceptions>
+		<exception>DuplicateRemoteAppEntry</exception>
```

Comment:
We have a convention to name the exceptions thrown because of a duplicated external reference code as Duplicate{EntityName}ExternalReferenceCodeException, so I think it should be DuplicateRemoteAppEntryExternalReferenceCodeException